packages/builder/src/lib/account.ts

```diff
@@ -0,0 +1,46 @@
+import { Account, mnemonicToSecretKey } from "algosdk";
+import * as fs from "fs";
+import YAML from "yaml";
+
+import { BuilderError } from "../internal/core/errors";
+import { ERRORS } from "../internal/core/errors-list";
+import type { AccountDef, HDAccount, MnemonicAccount } from "../types";
+
+export function mkAccounts (input: AccountDef[]): Account[] {
+  const accounts: Account[] = [];
+  for (const i of input) {
+    if ((i as HDAccount).path) {
+      throw new BuilderError(ERRORS.ACCOUNT.HD_ACCOUNT, {});
```

Comment:
Can you add a pubkey/path to the error object and message?

Reply:
OK